README.md

```diff
@@ -90,7 +90,7 @@ backend. Installing these tools adds a bit to initramfs and may slow down your
 host boot, but are needed if you'd like to use a particular backend:
 
     sudo apt-get install lvm2 thin-provisioning-tools
-    sudo apt-get install btrfs-tools
+    sudo apt-get install btrfs-tools squashfs-tools
```

Comment:
So the only part of README.md where it should be listed is the build from source part.

All other cases assume lxd was installed from a package, in which case squashfs-tools will have been brought in that way.